docs/management_api.md

```diff
@@ -285,6 +285,110 @@ curl http://localhost:8081/models/noop/all
 ]
 ```
 
+`GET /models/{model_name}/{model_version}?customized=true`
+or 
+`GET /models/{model_name}?customized=true`
+
+Use the Describe Model API to get detail runtime status and customized metadata of a version of a model:
+* Implement function describe_handle. Eg.
+```
+    def describe_handle(self):
+        """Customized describe handler
+        Returns:
+            dict : A dictionary response.
+        """
+        output_describe = None
+
+        logger.info("Collect customized metadata")
+
+        return output_describe
+```
+
+* Implement function _is_describe if handler is inherited from BaseHandler.
```

Comment:
@lxning , I think it should be if handler is **NOT** inherited from BaseHandler

Reply:
We should make sure we have documented that, it is recommended for custom handlers to extend the BaseHandler